assets/stylesheets/bootstrap/mixins/_gradients.scss

```diff
@@ -3,39 +3,45 @@
 // Horizontal gradient, from left to right
 //
 // Creates two color stops, start and end, by specifying a color and position for each color stop.
+// Color stops are not available in IE9 and below.
 @mixin gradient-horizontal($start-color: #555, $end-color: #333, $start-percent: 0%, $end-percent: 100%) {
   background-image: -webkit-linear-gradient(left, $start-color $start-percent, $end-color $end-percent); // Safari 5.1-6, Chrome 10+
   background-image: -o-linear-gradient(left, $start-color $start-percent, $end-color $end-percent); // Opera 12
-  background-image: linear-gradient(to right, $start-color $start-percent, $end-color $end-percent); // Standard, Firefox 16+, Opera 12.10+, Safari 7+, Chrome 26+
+  background-image: linear-gradient(to right, $start-color $start-percent, $end-color $end-percent); // Standard, IE10, Firefox 16+, Opera 12.10+, Safari 7+, Chrome 26+
+  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#{ie-hex-str($start-color)}', endColorstr='#{ie-hex-str($end-color)}', GradientType=1); // IE9 and down
```

Comment:
All changes in SASS, like this one, seem to have been done manually in https://github.com/entreprise7pro/bootstrap-sass/commit/588b616e94c9604cfe1dc5b2bb29e7dabef4642d
With the rake convert process, any of these changes need to be reapplied first in entreprise7pro/bootstrap (less files), then rake convert again in bootstrap-sass.

Reply:
Not sure if I understand, what is a rake convert?  normal browsers do not need a Microsoft.gradient

Even MS Edge no longer uses this.  MS Edge is based on Chromium now.

Reply:
Overall the changes look legit , except the .scss changes, please clean up the mentions of IE8 / 9 / 10 /10+ / 11.